py/private/py_binary.bzl

```diff
@@ -7,7 +7,6 @@ load("//py/private:utils.bzl", "PY_TOOLCHAIN", "SH_TOOLCHAIN", "dict_to_exports"
 load("//py/private/venv:venv.bzl", _py_venv = "py_venv_utils")
 
 def _py_binary_rule_imp(ctx):
-    bash_bin = ctx.toolchains[SH_TOOLCHAIN].path
```

Comment:
feels like we should leave a comment - I'm not that confident we've got it correct yet. this might just be a workaround for RBE not having the bash toolchain defined.